### PR TITLE
[KYUUBI #2156][FOLLOWUP] Fix configuration format in document

### PR DESCRIPTION
**Pull request by jiaoqingbo**
### _Why are the changes needed?_

fix #2157 cofiguration format error

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/security/hadoop_credentials_manager.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ The necessary configurations for hdfs and hive to obtain delegation token are as
...
-<div style='width: 40pt;word-wrap: break-word;white-space: normal'>hive.metastore.kerberos.principal</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>The service principal for the metastore thrift server. The special string _HOST will be replaced automatically with the correct host name.</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>for example hive/_HOST@${realm}</div>
-<div style='width: 40pt;word-wrap: break-word;white-space: normal'>hive.metastore.kerberos.keytab.file</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>The path to the Kerberos Keytab file containing the metastore thrift server's service principal.</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>for example /etc/security/keytabs/hive.service.keytab</div>
\ No newline at end of file
+hadoop\.security<br>\.authentication|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>Set the authentication for the cluster</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>kerberos</div>
```
Comment (pan3793) [inline docs/security/hadoop_credentials_manager.md:86]: let's quote it by backquote "\`" `hadoop\.security<br>\.authentication` or `<code>hadoop.security.authentication</code>`

Comment (pan3793) [reply docs/security/hadoop_credentials_manager.md:86]: See more information in previous discussion https://github.com/apache/incubator-kyuubi/pull/1890

**Comment by pan3793** [comment]
If possible, when you touch the document, would you please build the document in local and paste the previous screenshot in your PR description?

File: docs/security/hadoop_credentials_manager.md (hunk 1 above)
Comment (jiaoqingbo) [reply docs/security/hadoop_credentials_manager.md:86]: > 

It seems that the previous part of this document is also not modified according to #1890

Comment (jiaoqingbo) [reply docs/security/hadoop_credentials_manager.md:86]: > If possible, when you touch the document, would you please build the document in local and paste the previous screenshot in your PR description?

Ok ,I will paste

Comment (pan3793) [reply docs/security/hadoop_credentials_manager.md:86]: > It seems that the previous part of this document is also not modified according to #1890

Let's fix it as much as possible

Comment (cxzl25) [reply docs/security/hadoop_credentials_manager.md:86]: > It seems that the previous part of this document is also not modified according to #1890

Sorry I didn't notice this document when I changed it.
There are two more.
`authentication.md`
`kinit.md`

**Comment by jiaoqingbo** [comment]
the previous page 
image

**Comment by jiaoqingbo** [comment]
after changed
image
image
image
image

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging master/branch-1.5

**Pull request closed without merging**